### ISSUE TITLE
Search text box needs label for a11y

### ISSUE THREAD

**Issue by greatislander** [pressbooks/pressbooks-book#9]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all other plugins deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing this issue with a book using this theme?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks) to see if your bug or enhancement is already listed?

### Description

The new "search within" textbox should have a label - e.g. "search within this book."

### Steps to Reproduce

Not applicable

### System Information

Book Theme: Pressbooks Book 1.8.0
Pressbooks: 4.0.0-dev
WordPress: 4.8.0
PHP: 7.1
